### ISSUE TITLE
[fr] scroll list horizonal option

### ISSUE THREAD

**Issue by cubakcabak** [UnevenSoftware/LeafPic#435]

**Comment by rupak0577** [comment]
Can you explain it a bit? I don't see where a horizontal scroll would be required.

**Comment by dnldsht** [comment]
@rupak0577 I think he was talking about the grid of media or albums

**Comment by rupak0577** [comment]
@dnldsht Yes, but should we though? Because it makes sense to have a horizontal scroll only in landscape mode but not otherwise.

**Comment by dnldsht** [comment]
Personally I don't like that feature, I found it confusing, so I'm not
gonna implement it, but anybody can do that if it's disabled by default

**Comment by VinamraGuptaa** [comment]
Hey. Is this issue is still open? I would like to work on it.

**Comment by ilihas786** [comment]
can you explain the issue please i am beginner to open source
